File: debot-no-op/src/main/java/com/tomoima/debot/DebotConfigurator.java
package com.tomoima.debot;

import android.content.Context;

import com.tomoima.debot.strategy.DebotStrategy;

import java.util.ArrayList;

public class DebotConfigurator {

    public static void configureWithDefault(Context context) {

    }

    public static void configureWithCustomizedMenu(Context context, ArrayList<DebotStrategy> customList){

    }

    private static ArrayList<DebotStrategy> createDefaultMenuConfig(Context context){
        return new ArrayList<DebotStrategy>();
    }
}

File: debot-no-op/src/main/java/com/tomoima/debot/DebotConfigurator.kt
package com.tomoima.debot

import android.content.Context
import com.tomoima.debot.strategy.DebotStrategy
import java.util.*

object DebotConfigurator {

    @JvmStatic fun configureWithDefault() {

    }

    @JvmStatic fun configureWithCustomizedMenu(customList: ArrayList<DebotStrategy>) {

    }

    private fun createDefaultMenuConfig(context: Context): ArrayList<DebotStrategy> {
        return ArrayList()
    }
}
